docs/guides/mobile-plugin-ima-android.md

```diff
@@ -17,7 +17,7 @@ This plugin simplifies integration with Google Interactive Media Ads (IMA) for a
 
 <!-- As of 2023-07-15 -->
 
-This documentation is for the UID2 IMA Plugin for Android version 0.1.0.
+This documentation is for the UID2 IMA Plugin for Android version 0.4.0.
```

Comment:
Is it strange that this says 0.4.0 but the links refer to 0.5.0. We could say:

>This documentation is for the UID2 IMA Plugin for Android version 0.4.0+. 

This would suggest that it's compatible with 0.4.0 and upwards.

Reply:
True and thx! Updated that plus the two examples as discussed.